web/public/debug.log

```diff
@@ -0,0 +1,47 @@
+[0909/141219.091:ERROR:crash_report_database_win.cc(469)] failed to stat report
```

Comment:
Remover logs desnecessários.
Se esse arquivo for criado automáticamente, seria interessante deixar no .gitignore, não?